### PR TITLE
Document the 'netplan try' command

### PR DESCRIPTION
**Pull request by slyon**
## Done

Describe the `netplan try` command alongside `netplan generate` and `netplan apply`.

## QA

Executed `./run` and verified that the `netplan try` command is described in the "Commands" section on the frontpage.

### CODE REVIEW

**Comment by slyon** [comment]
I am a collaborator of the upstream netplan repo: https://github.com/CanonicalLtd/netplan

File: index.html [hunk 1]
```diff
@@ -65,6 +65,7 @@ <h3>Commands</h3>
       <ul>
         <li><strong>netplan generate</strong>: Use <code>/etc/netplan</code> to generate the required configuration for the renderers.</li>
         <li><strong>netplan apply</strong>: Apply all configuration for the renderers, restarting them as necessary.</li>
+        <li><strong>netplan try</strong>: Apply configuration and wait for user confirmation. Roll back if network is broken or no confirmation is given.</li>
```
Comment (hatched) [inline index.html:68]: Thoughts?
```suggestion
        <li><strong>netplan try</strong>: Apply configuration and wait for user confirmation; will roll back if network is broken or no confirmation is given.</li>
```

**Comment by anthonydillon** [comment]
@hatched can you release this site and add the label, please.

**Comment by hatched** [comment]
It was released yesterday right after it landed. I'll add a label.

**Pull request merged**